src/schema/posts.ts

```diff
@@ -449,6 +449,9 @@ export const reportReasons = new Map([
 export const getPostPermalink = (post: Pick<GQLPost, 'shortId'>): string =>
   `${process.env.URL_PREFIX}/r/${post.shortId}`;
 
+export const getPostPageLink = (post: Pick<GQLPost, 'shortId'>): string =>
```

Comment:
I think we have it already somewhere in the codebase

Reply:
commentsPermalink I think from the top of my head

Reply:
You're right, Chris! Thank you for the reference!